distribution/ddf-common/src/main/resources/etc/artemis.xml

```diff
@@ -54,7 +54,7 @@
         <network-check-URL-list>http://www.apache.org</network-check-URL-list> -->
 
         <network-check-period>10000</network-check-period>
-        <network-check-timeout>255</network-check-timeout>
+        <network-check-timeout>1000</network-check-timeout>
```

Comment:
revert this change linux doesn't support a timeout this large for the network check, can you also add a comment here that 255 is the max support by linux distro's for me.

Reply:
fixed